101-cloud-shell-vnet/azuredeploy.json

```diff
@@ -22,6 +22,7 @@
         },
         "contributorOID": {
```

Comment:
We should rename this.... 
1) it's a roleDefinitionId, not an OID
2) if it can be something other than contributor, we should name accordingly, if not, just make it a variable (less margin for error)

Reply:
this still needs to be done...